infra/main.bicep

```diff
@@ -576,8 +638,8 @@ module searchReaderRoleBackend 'core/security/role.bicep' = if (useAuthenticatio
   }
 }
 
-// For computer vision access by the backend
-module cognitiveServicesRoleBackend 'core/security/role.bicep' = if (useGPT4V) {
+// For computer vision or document intelligence access by the backend
+module cognitiveServicesRoleBackend 'core/security/role.bicep' = if (useGPT4V || useUserUpload) {
```

Comment:
Would they need this still if they used the local pdf parser?

Reply:
No, but currently we *always* create the Document Intelligence resource, even when using local parsers. It doesn't cost money to create it, just to use it. So our Bicep doesnt use the USE_LOCAL* to decide anything about provisioning.

Reply:
Resolving this, please re-open if this is a concern.